WalletWasabi/Models/ChaumianCoinJoin/CcjRound.cs

```diff
@@ -554,14 +555,14 @@ public async Task ExecuteNextPhaseAsync(CcjRoundPhase expectedPhase, Money feePe
 									}
 									catch (Exception ex)
 									{
-										Logger.LogWarning<CcjRound>($"Round ({RoundId}): {expectedPhase.ToString()} timeout failed with exception: {ex}.");
+										Logger.LogWarning($"Round ({RoundId}): {expectedPhase.ToString()} timeout failed with exception: {ex}.");
```

Comment:
Bellow in this same file (and in other places too) there is different pattern that I would prefer consisting on using two different log entries:

```suggestion
										Logger.LogWarning($"Round ({RoundId}): {expectedPhase.ToString()} timeout failed.");
										Logger.LogWarning(ex);
```